zetaclient/chains/bitcoin/rpc/rpc.go

```diff
@@ -157,3 +164,60 @@ func GetRecentFeeRate(rpcClient interfaces.BTCRPCClient, netParams *chaincfg.Par
 	// #nosec G115 always in range
 	return uint64(highestRate), nil
 }
+
+// CheckRPCStatus checks the RPC status of the evm chain
+func CheckRPCStatus(
+	client interfaces.BTCRPCClient,
+	alertLatency uint64,
+	tssAddress btcutil.Address,
+	logger zerolog.Logger,
+) error {
```

Comment:
I would keep the parameter as is. The `CheckRPCStatus` is not only intended to check latency only, it could check any necessary APIs used by zetaclient to print warnings (logs), for instance, get transaction, get receipt, get gas price.

Name conversion wise, Solana chain use the name `HEALTH_CHECK_SLOT_DISTANCE`, I wanted a unified terminology for config file and `CheckRPCStatus` method.

For RPC latency detection:
Without another trusted node to compare with (by latest height), the only way we could tell is how much time elapsed since latest block.

Reply:
My biggest concern is that we have too many args and logic within each function call. If this is an rpc package it should implement RP**calls**, metrics and other logic is out of the scope IMO